packages/trip-form/src/DateTimeSelector/index.tsx

```diff
@@ -137,6 +130,26 @@ export default function DateTimeSelector({
   timeZone = getUserTimezone()
 }: DateTimeSelectorProps): ReactElement {
   const intl = useIntl()
+  const baseColor = S.baseColor()
+
+  const departureOptions: DepartArriveOption[] = [
+    {
+      // Default option.
+      type: "NOW",
+      text: intl.formatMessage({ id: "otpUi.DateTimeSelector.now" })
+    },
+    {
+      type: "DEPART",
+      text: intl.formatMessage({ id: "otpUi.DateTimeSelector.depart" })
+    },
+    {
+      type: "ARRIVE",
+      text: intl.formatMessage({ id: "otpUi.DateTimeSelector.arrive" })
+    }
+  ];
+  departureOptions.forEach(opt => {
+    opt.isSelected = departArrive === opt.type;
+  });
```

Comment:
This works but it's pretty inefficient... Anyway we can replace/generate `isSelected` when we're iterating through the array anyway?